doc/ug/introduction.tex

```diff
@@ -321,6 +321,19 @@ \section{Requirements}
   version 1.2).
 \end{description}
 
+\subsection{Installing Requirements on Ubuntu 16.04 LTS}
+\label{sec:installing_requirements}
+\index{installing_requirements}
+
+To make ESPResSo run on Ubuntu 16.04 LTS its dependencies can be installed with:
```

Comment:
There are differences, because on the older ubuntus and debians cython is not new enough and has to be install via pip. 16.04 ships cython 0.23.4